README.md

```diff
@@ -102,6 +102,8 @@ Eslint was used to check for erroes in menu.js
 - **[Responsive Version]**
 - **[interactive hamburger menu]**
 - **[modal pop up on clicking see project]**
+- **[Form validation ]**
+
 
 
 
```

Comment:
- [ ] OPTIONAL
MIT license link in your ReadMe is broken, kindly endeavor to fix it, and remove the guide note below it, from _line 81 to 84_.